test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -842,12 +842,23 @@ public async Task HttpTrigger_Identities_BlocksSpoofedEasyAuthIdentity()
             }
         }
 
+        [Fact]
+        public async Task TestHostStatus_ContainsInstaceIdAndPlatformVersion()
```

Comment:
There's an existing test HostStatus_AdminLevel_Succeeds that tests all the properties. Just add this there instead